waku/waku_test.go

```diff
@@ -1629,7 +1685,9 @@ func TestRateLimiterIntegration(t *testing.T) {
 func TestMailserverCompletionEvent(t *testing.T) {
 	w1 := New(nil, nil)
 	require.NoError(t, w1.Start(nil))
-	defer w1.Stop() // nolint: errcheck
+	defer func() {
+		handleError(t, w1.Stop())
+	}()
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)